examples/C++/DDS/BasicConfigurationExample/BasicConfiguration_main.cpp

```diff
@@ -177,6 +177,15 @@ int main(
                     break;
             }
         }
+        if (transient && !reliable)
+        {
+            std::cerr << "WARNING: --transient will take no effect since not reliable." << std::endl;
+        }
+        if (type == SUBSCRIBER && count > 0 && transient && reliable)
+        {
+            count = 0;
+            std::cerr << "WARNING: -s | --samples will take no effect when transient local durability." << std::endl;
```

Comment:
This is not true. If samples is set, it will take the last N samples and exit.
The only thing is that while loop should stop when stopped.

Reply:
:+1: